### PR TITLE
Added/fixed random things in the guide

### PR DESCRIPTION
**Pull request by amsardesai**
- Corrected formatting errors
- Added more information about SF/California
- Added information about apps I used while I was in the US
- Added more info about credit cards
- Moved some sections to better locations (such as the blurb about Uber).

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -119,10 +119,11 @@ unless you have the financial ability to not care. Rent in major cities like SF
 
 <section id="Jobs-Perks" />
 ### Perks
-Some companies have cool perks like food, bikes, transit passes, happy hour,
+Some companies have cool perks like catered lunches/dinners, bikes, transit passes, happy hour,
 snacks, game rooms, pool tables, ping pong, foosball, museum passes, etc.
 
-It's worth checking out.
+It's worth checking out. Catered lunches/dinners can drastically help reduce your daily spending,
```
Comment (sholiday) [inline index.markdown]: Can you remove the line catered from this line and the above line. Companies have different ways of giving us free lunches and dinners.

File: index.markdown [hunk 2]
```diff
@@ -135,10 +136,13 @@ start date, and if they're using Cultural Vistas as a visa sponsor. It's useful
 for the visa. If your company isn't using Cultural Vistas, good luck, and be
 ready to provide a lot of school-related paperwork such as proof of enrolment.
 
-CECA and Cultural Vistas will guide you through the process.
+CECA and Cultural Vistas will guide you through the process. If you're an international
+student (i.e your passport isn't Canadian), this process will involve making a trip
```
Comment (sholiday) [inline index.markdown]: s/If you're an international student (i.e your passport isn't Canadian),/If you are not a Canadian citizen/

File: index.markdown [hunk 3]
```diff
@@ -135,10 +136,13 @@ start date, and if they're using Cultural Vistas as a visa sponsor. It's useful
...
+to the US consulate in Toronto for an interview.
```
Comment (sholiday) [inline index.markdown]: It doesn't have to be Toronto. I know people who went to Ottawa because of the wait.

File: index.markdown [hunk 4]
```diff
@@ -751,14 +766,28 @@ If you are traveling on a Canadian Passport to the US to visit (not J-1), you ar
...
+  vehicles instead of a SUV. You can request an Uber via SMS, or from their
+  iPhone or Android app. The seats are extremely comfortable.
+* [Venmo](http://venmo.com) is a popular US-only service for giving money to your
+  friends, similar to PayPal. It connects to your bank account and saves you the
+  effort of withdrawing cash by just using their mobile app. It's also completely
+  free and allows you to charge your friends. Ankit finds it very useful when
+  splitting things like restaurant bills between friends.
+* [Splitwise](http://splitwise.com): Allows you to keep track of expenses you made
+  for other people, and tracks who owes how much money. Ankit likes to use this
+  alongside Venmo.
```
Comment (sholiday) [inline index.markdown:789]: These additions are awesome!

File: index.markdown [hunk 5]
```diff
@@ -817,9 +845,13 @@ If you are an intern and have more info let me know.
 	W2015.
 
 <section id="Area-SF" />
-### San Francisco
+### San Francisco, CA
 * Since SF is a city, prices are pretty high.
-* It's very tough to find housing in the city, especially for cheap.
+* It's very tough to find housing in the city, especially for cheap. No, seriously.
+  The average rent in San Francisco is
```
Comment (sholiday) [inline index.markdown]: Average is a misleading metric, can you just link to the article but not state the average?

File: index.markdown [hunk 6]
```diff
@@ -874,32 +900,65 @@ If you are an intern and have more info let me know.
...
     and undoubtedly a great place to meet technical folks and entrepreneurs.
-
 * If you would rather live in a more student-populated area, Berkeley is an
   option. The commute to the downtown area is 15-20 min, but is worth it if you
   want to meet people your age.
 
-### Food
+#### Neighbourhoods
+* Tenderloin - There's some great Indian and African food here, but it is by far the sketchiest
+  neighbourhood in SF. Hobos everywhere, and it smells like shit.
```
Comment (sholiday) [inline index.markdown]: Please remove the comment about Hobo's and shit. Or say it in nice tone.

File: index.markdown [hunk 7]
```diff
@@ -933,22 +991,42 @@ It's worth a trip after your first paycheque.
...
+    Beach, where all the nightlife is.
+
 * Las Vegas
   * Cirque de Soleil shows are a must.
   * Grand Canyon if you have a day or two. Drive out is about 4 hours each way
     however so be warned.
   * Hoover Dam has a nice indoor tour where you get to see the generators and
     water tunnels. Also a beautiful sight.
-  * Casinos and restaurants of course on the strip.
+  * Casinos and restaurants of course on the strip, for those 21 and over.
```
Comment (sholiday) [inline index.markdown]: I don't think most restaurants are 21+?

File: index.markdown (hunk 6 above)
Comment (amsardesai) [inline index.markdown]: My bad, will remove! Got a bit passionate about the Tenderloin there :P

File: index.markdown (hunk 7 above)
Comment (amsardesai) [inline index.markdown]: I'll remove it altogether, I think the 21 or older part is pretty self explanatory.

File: index.markdown (hunk 3 above)
Comment (amsardesai) [inline index.markdown]: Done!

**Comment by amsardesai** [comment]
@sholiday Addressed review comments!

**Comment by amsardesai** [comment]
@sholiday bump.

**Comment by sholiday** [comment]
Hey sorry I've been so slow. I've added @tonyd3 as a committer and he can do the review.

**Comment by tonyd3** [comment]
lgtm

**Pull request merged**